src/components/pokemon/Pokemon.js

```diff
@@ -0,0 +1,48 @@
+import React, {useEffect, useState} from 'react';
+import axios from "axios";
+import './Pokemon.css';
+
+function Pokemon({pokemonURL}) {
+    const [pokemonState, setPokemonState] = useState({});
+
+    useEffect(() => {
+        async function fetchPokemon() {
+            try {
+                const {data} = await axios.get(`${pokemonURL}`);
+                console.log(data);
+                setPokemonState(data);
+            } catch (e) {
+                console.log(e)
+            }
+        }
+
+        fetchPokemon();
+    }, [])
+
```

Comment:
Als je hier
```suggestion
    if (!pokemonState) return null;
```
doet, dan kun je de check die hieronder staat weghalen en ben je wat nesting kwijt